tdrs-backend/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.10.1-slim-buster
+FROM python:3.10.4-slim-buster
```

Comment:
Required to get everything to run, for some reason. i think this has already been committed to develop